src/cryptoadvance/specter/templates/includes/tx-row.html

```diff
@@ -106,15 +106,19 @@
             <span class="amount-price note hidden">()</span>
         </td>
         <td id="column-time" class="time"></td>
-        <td id="column-confirmations" >
-            <span class="confirmations"></span>
-            <button class="rbf btn optional hidden" style="width: 130px; float: right;" type="button">{{ _("Speed up") }}</button>
-            <button class="rbf-cancel danger btn optional hidden" style="width: 130px; float: right; margin-right: 10px;" type="button">{{ _("Cancel transaction") }}</button>
+        <td id="column-confirmations">
+            <div style="display: flex;">
+                <span class="confirmations"></span>
+            </div>
         </td>
         <td class="hidden optional blockhash"></td>
         <td id="column-action" class="highlight-box">
```

Comment:
I have a small doubt, isn't this highlight-box supposed to show some specific element? If so what is it, and when that element is visible, will it affect our current button's positions?